src/tests/universal-nrp-e2e/StartLocalTest.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# StartLocalTest.sh
+# Description: This script orchestrates tests on a local machine. Installs dependencies, 
+#              runs tests, and collects logs/reports. Returns an error code if any stage fails.
+#
+# Usage: ./StartLocalTest.sh [-s stage-name] [-p policy-package.zip -c resource-count [-r] [-g]]
```

Comment:
>Usage: ./StartLocalTest.sh [-s stage-name] [-p policy-package.zip -c resource-count [-r] [-g]] [](http://example.com/codeflow?start=2&length=94)

We need the option for the simple script invocation, without any arguments. Can you please also update this comment here?

Reply:
The simple script invocation is with `-p policy-package.zip -c resource-count`, it requires a policy package and instance count.

Reply:
Can we skip the resource 'instance' count and have the test pull that out from the package?

Reply:
Its not that straightforward, i would rather make it as a follow-up PR.